docs/Android.md

```diff
@@ -146,10 +138,7 @@ commands to copy the Swift Android stdlib:
 
 ```
 $ adb push build/Ninja-ReleaseAssert/swift-linux-x86_64/lib/swift/android/libswiftCore.so /data/local/tmp
-$ adb push build/Ninja-ReleaseAssert/swift-linux-x86_64/lib/swift/android/libswiftGlibc.so /data/local/tmp
 $ adb push build/Ninja-ReleaseAssert/swift-linux-x86_64/lib/swift/android/libswiftSwiftOnoneSupport.so /data/local/tmp
-$ adb push build/Ninja-ReleaseAssert/swift-linux-x86_64/lib/swift/android/libswiftRemoteMirror.so /data/local/tmp
```

Comment:
This one is probably not needed for the hello world, but if you end up try to reflect on the application, you will need this.

Reply:
I wasn't sure if it was worth keeping these two libraries since the "hello world" example doesn't need them, but if you think they will be needed generally, I'll add them back.